package.json

```diff
@@ -231,6 +233,7 @@
     "content-security-policy-parser": "^0.1.0",
     "cookie": "^0.3.1",
     "css-loader": "^0.28.3",
+    "css-raw-loader": "^0.1.2",
```

Comment:
I think requiring it only once is enough, isn't it?

Reply:
Also yarn outputs a warning:

```
warning package.json: "dependencies" has dependency "css-raw-loader" with range "0.1.2" that collides with a dependency in "devDependencies" of the same name with version "^0.1.2"
```

Reply:
Sorry about that! Fixed.